pyproject.toml

```diff
@@ -103,19 +105,22 @@ pytest-xprocess = "^0.20.0"
 MarkupSafe = "2.1.1"
 pytest-snapshot = "^0.9.0"
 channels = "^3.0.5"
+python-multipart = "^0.0.5"
+pygments = "^2.3"
 
 [tool.poetry.extras]
 aiohttp = ["aiohttp", "pytest-aiohttp"]
-asgi = ["starlette"]
-debug-server = ["starlette", "uvicorn"]
+asgi = ["starlette", "python-multipart"]
+debug-server = ["starlette", "uvicorn", "python-multipart", "click", "pygments"]
 django = ["Django", "pytest-django", "asgiref"]
 channels = ["channels", "asgiref"]
 flask = ["flask", "pytest-flask"]
 opentelemetry = ["opentelemetry-api", "opentelemetry-sdk"]
 pydantic = ["pydantic"]
 sanic = ["sanic"]
-fastapi = ["fastapi"]
+fastapi = ["fastapi", "python-multipart"]
 chalice = ["chalice"]
+cli = ["click", "pygments"]
```

Comment:
I think it is fine to keep pygments here, I think in future we'll combine cli and debug-server, I'm thinking of making our cli more powerful 😊